### ISSUE TITLE
Crash trying to open the message composer without internet connection

### ISSUE THREAD

**Issue by bergezps** [intercom/intercom-android#236]
## Version info
- Intercom for Android version: io.intercom.android:intercom-sdk-base:3.+
- Android OS version: 6.0.1
- Android Device: Nexus 5
## Expected behavior

Show an internet connection problem message
## Actual behavior

Crash
## Steps to reproduce
1.  Turn off internet connection
2. Open the message composer
   Intercom.client().displayMessageComposer();
## Logs

```
                                                                          at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
                                                                           at android.support.design.widget.Snackbar.<init>(Snackbar.java:190)
                                                                           at android.support.design.widget.Snackbar.make(Snackbar.java:217)
                                                                           at android.support.design.widget.Snackbar.make(Snackbar.java:242)
                                                                           at io.intercom.android.sdk.conversation.ConversationFragment$9.handleMessage(ConversationFragment.java:446)
                                                                           at android.os.Handler.dispatchMessage(Handler.java:98)
                                                                           at android.os.Looper.loop(Looper.java:148)
                                                                           at android.app.ActivityThread.main(ActivityThread.java:5417)
                                                                           at java.lang.reflect.Method.invoke(Native Method)
                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
                                                                        Caused by: android.view.InflateException: Binary XML file line #18: Error inflating class android.support.design.widget.Snackbar$SnackbarLayout
                                                                           at android.view.LayoutInflater.createView(LayoutInflater.java:645)
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764)
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
                                                                           at android.support.design.widget.Snackbar.<init>(Snackbar.java:190) 
                                                                           at android.support.design.widget.Snackbar.make(Snackbar.java:217) 
                                                                           at android.support.design.widget.Snackbar.make(Snackbar.java:242) 
                                                                           at io.intercom.android.sdk.conversation.ConversationFragment$9.handleMessage(ConversationFragment.java:446) 
                                                                           at android.os.Handler.dispatchMessage(Handler.java:98) 
                                                                           at android.os.Looper.loop(Looper.java:148) 
                                                                           at android.app.ActivityThread.main(ActivityThread.java:5417) 
                                                                           at java.lang.reflect.Method.invoke(Native Method) 
                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
                                                                        Caused by: java.lang.reflect.InvocationTargetException
                                                                           at java.lang.reflect.Constructor.newInstance(Native Method)
                                                                           at android.view.LayoutInflater.createView(LayoutInflater.java:619)
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764) 
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704) 
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:492) 
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
                                                                           at android.support.design.widget.Snackbar.<init>(Snackbar.java:190) 
                                                                           at android.support.design.widget.Snackbar.make(Snackbar.java:217) 
                                                                           at android.support.design.widget.Snackbar.make(Snackbar.java:242) 
                                                                           at io.intercom.android.sdk.conversation.ConversationFragment$9.handleMessage(ConversationFragment.java:446) 
                                                                           at android.os.Handler.dispatchMessage(Handler.java:98) 
                                                                           at android.os.Looper.loop(Looper.java:148) 
                                                                           at android.app.ActivityThread.main(ActivityThread.java:5417) 
                                                                           at java.lang.reflect.Method.invoke(Native Method) 
                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
                                                                        Caused by: android.view.InflateException: Binary XML file line #20: Binary XML file line #20: Error inflating class TextView
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
                                                                           at android.support.design.widget.Snackbar$SnackbarLayout.<init>(Snackbar.java:686)
                                                                           at java.lang.reflect.Constructor.newInstance(Native Method) 
                                                                           at android.view.LayoutInflater.createView(LayoutInflater.java:619) 
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764) 
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704) 
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:492) 
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
                                                                           at android.support.design.widget.Snackbar.<init>(Snackbar.java:190) 
                                                                           at android.support.design.widget.Snackbar.make(Snackbar.java:217) 
                                                                           at android.support.design.widget.Snackbar.make(Snackbar.java:242) 
                                                                           at io.intercom.android.sdk.conversation.ConversationFragment$9.handleMessage(ConversationFragment.java:446) 
                                                                           at android.os.Handler.dispatchMessage(Handler.java:98) 
                                                                           at android.os.Looper.loop(Looper.java:148) 
                                                                           at android.app.ActivityThread.main(ActivityThread.java:5417) 
                                                                           at java.lang.reflect.Method.invoke(Native Method) 
                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
                                                                        Caused by: android.view.InflateException: Binary XML file line #20: Error inflating class TextView
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:782)
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
                                                                           at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:489)
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:374) 
                                                                           at android.support.design.widget.Snackbar$SnackbarLayout.<init>(Snackbar.java:686) 
                                                                           at java.lang.reflect.Constructor.newInstance(Native Method) 
                                                                           at android.view.LayoutInflater.createView(LayoutInflater.java:619) 
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764) 
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704) 
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:492) 
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
                                                                           at android.support.design.widget.Snackbar.<init>(Snackbar.java:190) 
                                                                           at android.support.design.widget.Snackbar.make(Snackbar.java:217) 
                                                                           at android.support.design.widget.Snackbar.make(Snackbar.java:242) 
                                                                           at io.intercom.android.sdk.conversation.ConversationFragment$9.handleMessage(ConversationFragment.java:446) 
                                                                           at android.os.Handler.dispatchMessage(Handler.java:98) 
                                                                           at android.os.Looper.loop(Looper.java:148) 
                                                                           at android.app.ActivityThread.main(ActivityThread.java:5417) 
                                                                           at java.lang.reflect.Method.invoke(Native Method) 
                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
                                                                        Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 6: TypedValue{t=0x2/d=0x101009b a=1}
                                                                           at android.content.res.TypedArray.getColorStateList(TypedArray.java:482)
                                                                           at android.widget.TextView.<init>(TextView.java:750)
                                                                           at android.widget.TextView.<init>(TextView.java:671)
                                                                           at android.support.v7.widget.AppCompatTextView.<init>(AppCompatTextView.java:60)
                                                                           at android.support.v7.widget.AppCompatTextView.<init>(AppCompatTextView.java:56)
                                                                           at android.support.v7.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:103)
                                                                           at android.support.v7.app.AppCompatDelegateImplV7.createView(AppCompatDelegateImplV7.java:1008)
                                                                           at android.support.v7.app.AppCompatDelegateImplV7.onCreateView(AppCompatDelegateImplV7.java:1067)
                                                                           at android.support.v4.view.LayoutInflaterCompatHC$FactoryWrapperHC.onCreateView(LayoutInflaterCompatHC.java:44)
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:746)
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704) 
                                                                           at android.view.LayoutInflater.rInflate(LayoutInflater.java:835) 
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:489) 
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:374) 
                                                                           at android.support.design.widget.Snackbar$SnackbarLayout.<init>(Snackbar.java:686) 
                                                                           at java.lang.reflect.Constructor.newInstance(Native Method) 
                                                                           at android.view.LayoutInflater.createView(LayoutInflater.java:619) 
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764) 
                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704) 
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:492) 
                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
                                                                           at android.support.design.widget.Snackbar.<init>(Snackbar.java:190) 
                                                                           at android.support.design.widget.Snackbar.make(Snackbar.java:217) 
                                                                           at android.support.design.widget.Snackbar.make(Snackbar.java:242) 
                                                                           at io.intercom.android.sdk.conversation.ConversationFragment$9.handleMessage(ConversationFragment.java:446) 
                                                                           at android.os.Handler.dispatchMessage(Handler.java:98) 
                                                                           at android.os.Looper.loop(Looper.java:148) 
                                                                           at android.app.ActivityThread.main(ActivityThread.java:5417) 
                                                                           at java.lang.reflect.Method.invoke(Native Method) 
                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
```

```
```

**Comment by ronocod** [comment]
What versions of the support libraries are you using in your app? This seems like a version conflict.

**Comment by ronocod** [comment]
There's a chance this is fixed in 3.0.4, let us know if using that version helps.

**Comment by ronocod** [comment]
Haven't been able to reproduce this, and we haven't had any reports of this in some time. Version 3.0.11 uses Support Library 25.0.0, if you can reproduce it with that then please let us know.
